db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.3.0_rollback.sql

```diff
@@ -1,23 +1,34 @@
 -- Rollback script for master.app_detail
+INSERT INTO master.app_detail SELECT * FROM master.app_detail_bkp;
```

Comment:
We will have 'eng' entries in th eapp_detail, so when this statement executes, will it not throw duplicate key error?
Have we tested this script?
@dhanendra06

Reply:
Tested the script it's not throwing the error for duplicate key